src/bin/dual-iir.rs

```diff
@@ -199,6 +214,7 @@ mod app {
         adcs: (Adc0Input, Adc1Input),
         dacs: (Dac0Output, Dac1Output),
         iir_state: [[iir::Vec5<f32>; IIR_CASCADE_LENGTH]; 2],
+        dds_clock_state: Option<ClockConfig>,
```

Comment:
This never actually changes in the code. Why do we store it as a `Resource`? It seems like this should just exist within `Pounder` instead.